rotkehlchen/tests/api/test_ethereum_transactions.py

```diff
@@ -1051,3 +1067,143 @@ def query_transactions(rotki, tx_module: EthTransactions) -> None:
     ):
         assert cursor.execute(f'SELECT COUNT(*) from {name}').fetchone()[0] == 0
     assert dbevents.get_history_events(HistoryEventFilterQuery.make(), True) == []
+
+
+def create_tx(tx_hash: bytes) -> EthereumTransaction:
```

Comment:
You know what would be nice? Add this as `make_ethereum_transaction()` here: https://github.com/rotki/rotki/blob/develop/rotkehlchen/tests/utils/factories.py

And you can even have the tx_hash as optional argument. If not given you can make it create a random tx hash

Reply:
I don't see you having done it :)